crate/multiworld-installer/src/main.rs

```diff
@@ -975,10 +1001,24 @@ impl Application for State {
                 None,
             ),
             Page::Elevated => (
-                Text::new("The installer has been reopened with admin permissions. Please continue there.").into(),
+                Text::new(self.locale.message(InstallerReopenUAC)).into(),
                 false,
                 None,
             ),
+            Page::SelectLocale { .. } => (
+                {
+                    let mut col = Column::new();
+                    col = col.push("Select language you wish to proceed with");
```

Comment:
This text should probably be kept as simple as possible, so someone with minimal knowledge of English can understand it:

```suggestion
                    col = col.push("Select language");
```